### PR TITLE
Graph open by default, legend closable and under graph

### PR DESCRIPTION
**Pull request by PeterPsota**
https://github.com/vacuumlabs/verejne.digital/issues/198
Graph shown by default
Legend moved under graph
Legend closable
Moved tutorial/controls of graph to separate component `GraphControls`
Legend under graph now contains `GraphControls`
image
image

### CODE REVIEW

File: client/src/components/shared/Legend/Legend.js [hunk 1]
```diff
@@ -4,6 +4,8 @@ import classnames from 'classnames'
 import CircleIcon from 'react-icons/lib/fa/circle-o'
 import FilledCircleIcon from 'react-icons/lib/fa/circle'
 import {compose, withState, withHandlers} from 'recompose'
+import {Row, Col} from 'reactstrap'
+import GraphControlls from '../../Connections/components/GraphControlls'
```
Comment (borisvida) [inline client/src/components/shared/Legend/Legend.js]: Probably a typo, causes failing to compile.

**Comment by PeterPsota** [comment]
@borisvida 
I agree. I was thinking we could have white open graph button when graph is closed and similarly styled 'x' close button in top right of graph when open.
Separate PR or continue here?

**Comment by borisvida** [comment]
@PeterPsota Please, continue here, it's too tightly coupled to split :slightly_smiling_face:

**Comment by PeterPsota** [comment]
Toggling graph is now more like in legend, looks better

**Comment by PeterPsota** [comment]
Merge master, now uses `SubgraphInstructions` instead of `GraphControls` as the components were duplicate

**Pull request merged**